presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/api/PackageAPI.java

```diff
@@ -74,12 +76,12 @@ public synchronized Response install(@ApiParam("Url to fetch package") String ur
             return Response.status(BAD_REQUEST).entity("Expected URL in the request body").build();
         }
         try {
-            if (isInstalled()) {
+            if (PACKAGE_TYPE == RPM && isInstalled()) {
```

Comment:
It is `final` though. You could only assign to it once. How would you check if it is changed to TARBALL later.